Ghidra/Features/Base/src/main/java/ghidra/program/flatapi/FlatProgramAPI.java

```diff
@@ -1514,6 +1514,58 @@ public final List<Symbol> getSymbols(String name, Namespace namespace) {
 	public final Namespace getNamespace(Namespace parent, String namespaceName) {
 		return currentProgram.getSymbolTable().getNamespace(namespaceName, parent);
 	}
+	
+	/**
+	 * Creates a new namespace with the given name contained inside the
+	 * specified parent namespace.
+	 * Pass <code>null</code> for namespace to indicate the global namespace.
+	 * If a Namespace or GhidraClass with the given name already exists, the
+	 * existing one will be returned.
+	 * @param parent the parent namespace, or null for global namespace
```

Comment:
Should mention that parent must belong to the currentProgram.  IllegalArgumentException thrown if specified parent is from a different program.  This applies to createClass as well.  The specified parent may also be a Library in both cases.

Reply:
You can actually create a Namespace in a Library. I feel like I may have misunderstood that last bit though.

![image](https://user-images.githubusercontent.com/46897303/184252014-4aa0527d-e69f-45a6-8b1b-9a0768c2a661.png)

Reply:
My point about the Library as a parent was just an FYI.  But the specified parent object must belong currentProgram or an exception will be thrown.  Thought you shuold document the `@throws IllegalArgumentException` for that case.

Reply:
At this point PR is good enough.  I will begin merge processing and add this missing restriction comment to javadocs.